hw/ip/rom_ctrl/dv/cov/rom_ctrl_cov_unr_excl.el

```diff
@@ -0,0 +1,71 @@
+// Copyright lowRISC contributors.
```

Comment:
@rswarbrick @prajwalaputtappa could you check if these two files correctly captured what you mentioned in the review meeting?

Reply:
Also with UNR (VCS auto-generated), there are two uncovered FSM transactions:
![image](https://user-images.githubusercontent.com/11466553/152436968-3b6add32-d885-46b8-9a3b-ef1da8faf881.png)
I did not exclude them yet in manual exclusion, could you confirm if they are truly unreachable.

Reply:
Thanks @cindychip!

Both of those transitions should be unreachable. The FSM, excluding the `Invalid` terminal state looks like this:
![image](https://user-images.githubusercontent.com/104845/152508020-e7e9a5ac-6c67-4ba7-a8ea-db75f8674cdf.png)

I'd be really interested to understand why the UNR tool understands that other transitions are impossible but not these ones, but I guess sometimes things will remain a mystery :-p

Reply:
Thanks for confirming! I will add that to another manually excluded UNR file.